### PR TITLE
Add experimental option to pipe attached audio tracks through webaudio API

### PR DESCRIPTION
**Pull request by lukasIO**

### CODE REVIEW

File: src/room/track/RemoteAudioTrack.ts [hunk 1]
```diff
@@ -48,9 +65,41 @@ export default class RemoteAudioTrack extends RemoteTrack {
...
     return element;
   }
 
+  /**
+   * @internal
+   */
+  setAudioContext(audioContext: AudioContext) {
+    this.audioContext = audioContext;
+    if (this.attachedElements.length > 0) {
+      this.setupWebAudio(audioContext, this.attachedElements[0]);
```
Comment (davidzhao) [inline src/room/track/RemoteAudioTrack.ts:85]: if attached to multiple elements, do we only need to handle the first one?

Comment (davidzhao) [reply src/room/track/RemoteAudioTrack.ts:85]: or would it be a limitation that it only handles a single audio element?

**Review by davidzhao — APPROVED** [review]
lg, I'll figure out a way to make testing audio a bit easier via the CLI.

File: src/room/track/RemoteAudioTrack.ts (hunk 1 above)
Comment (lukasIO) [reply src/room/track/RemoteAudioTrack.ts:85]: for webaaudio we only need one element, as we're actually not interested in the element, but rather in the mediastream attached with it. we wouldn't want to pipe the same audio track multiple times into web audio

Comment (lukasIO) [reply src/room/track/RemoteAudioTrack.ts:85]: but just realised detach logic is still missing...

Comment (davidzhao) [reply src/room/track/RemoteAudioTrack.ts:85]: good call!

**Pull request merged**